src/controllers/DappsStakingController.ts

```diff
@@ -111,8 +111,66 @@ export class DappsStakingController extends ControllerBase implements IControlle
+            /*
+                        #swagger.description = 'Retrieves dapps staking TVL for a given network and period.'
+                        #swagger.tags = ['Dapps Staking']
+                        #swagger.parameters['network'] = {
+                            in: 'path',
+                            description: 'The network name. Supported networks: astar',
+                            required: true,
+                            enum: ['astar']
+                        }
+                        #swagger.parameters['period'] = {
+                            in: 'path',
+                            description: 'The period type. Supported values: 1 day, 7 days, 30 days, 90 days, 1 year',
+                            required: true,
+                            enum: ['1 day', '7 days', '30 days', '90 days', '1 year']
+                        }
+                    */
+            res.json(
+                await this._dappsStakingEvents.getDappStakingTvl(
+                    req.params.network as NetworkType,
+                    req.params.period as PeriodType,
+                ),
+            );
+        });
+
+        app.route('/api/v3/:network/dapps-staking/stakerscount/:contractAddress/:period').get(
+            async (req: Request, res: Response) => {
+                /*
+                        #swagger.description = 'Retrieves dapps staking stakers count for a given network and period.'
+                        #swagger.tags = ['Dapps Staking']
+                        #swagger.parameters['network'] = {
+                            in: 'path',
+                            description: 'The network name. Supported networks: astar',
+                            required: true,
+                            enum: ['astar']
+                        }
+                        #swagger.parameters['contractAddress'] = {
+                            in: 'path',
+                            description: 'Contract address to get stats for',
+                            required: true
+                        }
+                        #swagger.parameters['period'] = {
+                            in: 'path',
+                            description: 'The period type. Supported values: 1 day, 7 days, 30 days, 90 days, 1 year',
+                            required: true,
+                            enum: ['1 day', '7 days', '30 days', '90 days', '1 year']
+                        }
+                    */
+                res.json(
+                    await this._dappsStakingEvents.getDappStakingStakersCount(
+                        req.params.network as NetworkType,
+                        req.params.contractAddress as string,
+                        req.params.period as PeriodType,
+                    ),
+                );
+            },
+        );
+
         app.route('/api/v1/:network/dapps-staking/earned/:address').get(async (req: Request, res: Response) => {
```

Comment:
I am missing an endpoint which will provide array of dApps data. I am mainly interested in current number of stakers, but you can provide all below fields.
 ```
contractAddress
registeredAt: BigInt!
registrationBlockNumber: Int!
unregisteredAt: BigInt
unregistrationBlockNumber: Int
stakersCount: Int!```

Reply:
Done!